be/src/vec/sink/vtablet_block_convertor.cpp

```diff
@@ -568,10 +569,15 @@ Status OlapTableBlockConvertor::_partial_update_fill_auto_inc_cols(vectorized::B
     for (size_t i = 0; i < rows; i++) {
         dst_values.emplace_back(_auto_inc_id_allocator.next_id());
     }
+    // in VRowDistribution we reuse _batching_block, so the input block may already include PARTIAL_UPDATE_AUTO_INC_COL,
```

Comment:
When you got here, which means the `block` object is from `_batching_block`, it already have __PARTIAL_UPDATE_AUTO_INC_COL__, so we don't need to call the method  `_partial_update_fill_auto_inc_cols()`
we'd better to change the logic at L80

Reply:
done